G1ANT.Addon.Selenium/Commands/SeleniumGetHtml.cs

```diff
@@ -0,0 +1,48 @@
+/**
+*    Copyright(C) G1ANT Ltd, All rights reserved
```

Comment:
shouldn't be "G1ANT Robot Ltd"?

Reply:
Should be changing everywhere. We need decision from Business how to do this.